src/index.html

```diff
@@ -22,6 +22,424 @@
+          alt="Moyo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion Header -->
+    <main class="main">
+      <div class="catalog">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          />
+
+          <h3 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <div class="card__code">
+            <span class="card__code-label">Product code:</span>
+            <span class="card__code-value">195434</span>
+          </div>
+
+          <div class="card__rating rating">
+            <div class="rating__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="rating__desc">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price-value">$2,199</div>
+          </div>
+
+          <a
+            class="card__link"
+            href="#"
+            data-qa="card-hover"
```

Comment:
The `data-qa` attribute for the 'Buy' link inside the first card should be `data-qa="card-hover"` instead of `data-qa="hover"`. Consistency in naming conventions is crucial for maintainability and testing.